azure-key-vault-properties-provider/2.1/modules/ROOT/pages/azure-key-vault-properties-provider-reference.adoc

```diff
@@ -50,6 +50,8 @@ Use the following environment variables to configure the behavior of the Azure K
 - `AZURE_TENANT_ID`
 - `MULE_AZURE_KEY_VAULT_NAME`
 
+These are stored in secure configuration files with the values encrypted. The values are decrypted by the runtime. After they are decrypted by the runtime, they remain in memory in their decrypted state. For more information, refer to xref:cloudhub-2::ch2-protect-app-props.adoc[].
```

Comment:
```suggestion
CloudHub 2.0 stores these variables in secure configuration files with the values encrypted. During execution, the runtime decrypts these values and they remain in memory in their decrypted state. For more information, refer to xref:cloudhub-2::ch2-protect-app-props.adoc[].
```
Not sure if CH2 and the phrasing is correct here, but try to switch to active voice if possible.